core/src/main/java/org/dozer/propertydescriptor/PropertyDescriptorFactory.java

```diff
@@ -49,8 +50,12 @@ public static DozerPropertyDescriptor getPropertyDescriptor(Class<?> clazz, Stri
         boolean isMapProperty = MappingUtils.isSupportedMap(clazz);
         if (name.equals(DozerConstants.SELF_KEYWORD) &&
             (mapSetMethod != null || mapGetMethod != null || isMapProperty)) {
```

Comment:
In XSD for `mapSetMethod` and `mapGetMethod` it is stated:
> This should only be used of Map backed objects

So it should be safe to remove these `!= null` checks and have only newly introduced `isBlank` verification.